src/vs/workbench/browser/parts/titlebar/windowTitle.ts

```diff
@@ -58,6 +61,7 @@ export class WindowTitle extends Disposable {
 		@IViewsService private readonly viewsService: IViewsService
 	) {
 		super();
+		this._titleIncludesFocusedView = this.configurationService.getValue<string>(WindowSettingNames.title)?.match(/\${focusedView}/) !== null;
```

Comment:
You are duplicating the same logic here and then in line 89. A pattern I typically use is to have a method for when to react to a configuration change event that has the logic and then also call that method once on construction time.